charts/metallb/templates/speaker.yaml

```diff
@@ -1,31 +1,131 @@
+{{- if .Values.frr.enabled }}
+# FRR expects to have these files owned by frr:frr on startup.
+# Having them in a ConfigMap allows us to modify behaviors: for example enabling more daemons on startup.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frr-startup
```

Comment:
ConfigMap `.metadata.name` should incorporate the release name using `{{ template "metallb.fullname" . }}-` like other resources do.  It should also include the standard set of labels, including `app.kubernetes.io/component`:

Example from speaker Daemonset:
```
apiVersion: apps/v1
kind: DaemonSet
metadata:
  name: {{ template "metallb.fullname" . }}-speaker
  labels:
    {{- include "metallb.labels" . | nindent 4 }}
    app.kubernetes.io/component: speaker
...
```

Reply:
thanks